CHANGELOG.md

```diff
@@ -1,10 +1,26 @@
 # Confluent's Golang client for Apache Kafka
 
+# v2.5.0
+
+This is a feature release.
+
+## Fixes
+
+ * Windows builds are linked to `libssp` in addition to other libraries, since
+   the bundled zstd and zlib are built with `-fstack-protector`, and not linking
+   causes build failures. (#1184).
```

Comment:
```suggestion
 * Issues: #965
   Windows builds are linked to `libssp` in addition to other libraries, since
   the bundled zstd and zlib are built with `-fstack-protector`, and not linking
   causes build failures.
   Happening since 2.0.0 (#1184).
```